cumulus/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "ingest-workflows",
-  "version": "0.0.0",
+  "name": "@cumulus/cumulus",
```

Comment:
@kkelly51 this is not a package and is just a legacy thing from the GitC time. could we remove the name here so it is not published by lerna and also unpublish it from npm?